web/src/features/weather-layers/hooks.ts

```diff
@@ -28,21 +28,23 @@ export function useInterpolatedData(
     );
     return null;
   }
-  console.info(
-    `#1 ${type} forecast target ${formatDistance(tBefore, new Date(), {
-      addSuffix: true,
-    })} made ${formatDistance(getReferenceTime(gribs1[0]), new Date(), {
-      addSuffix: true,
-    })}`
-  );
+  if (import.meta.env.DEV) {
+    console.debug(
+      `#1 ${type} forecast target ${formatDistance(tBefore, new Date(), {
```

Comment:
There might be a Intl built in equivalent of this if we don't want to remove it from production builds.